### ISSUE TITLE
Gridpoint visible

### ISSUE THREAD

**Issue by JoopN** [DarwinNE/FidoCadJ#211]
When I'm designing a new item for my lib then I'm starting with something from the other libs or from scratch. It would be a great help if you can see on what grid point your mouse is on. Now you get with right mouse button 0.0 from the ruler. Could there be an option that you get the grid point your mouse is on?

**Comment by DarwinNE** [comment]
Hello,
  I am not sure to understand what you mean. The coordinates that are shown next to the toolbar (the exact location may depend on the operating system you are using) is not enough?

![fcj](https://user-images.githubusercontent.com/11355790/164993189-a8298c03-185e-47a3-9cee-f94666b2ad30.png)

Regards,
D.

**Comment by JoopN** [comment]
True, but I use a pen on a tablet, so no fixed things and just set your eyes to the bar for the coordinates have the effect that you will move a little when on mm grid. With a big screen those coordinates are far away so to speak. When focussed on the spot you could click pen button (right click mouse) without moving. Made some rotary switches and setting the contacts right was sometimes one mm down or up or left or right in order to get the right picture. I made my lib in a text editor, not in FidoCadj. So FidoCadj open with object under construction and text editor next to it for adjusting and recording the coordinates on a Raspberry Pi 400 with OpenJDK 11.

**Comment by JoopN** [comment]
The coordinates do not update when you drag a line over your drawing area. Only when you stop its updated, but that is very annoying in the drawing process.

**Comment by DarwinNE** [comment]
I could reproduce the latter issue, indeed, it would be useful to update the information shown.
